cmd/thanos/query.go

```diff
@@ -753,17 +763,27 @@ func engineFactory(
 		if ld < r {
 			lookbackDelta = time.Duration(r) * time.Millisecond
 		}
-		engines[i] = newEngine(promql.EngineOpts{
+
+		var queryTracker *promql.ActiveQueryTracker
+		newEngingOpts := promql.EngineOpts{
```

Comment:
```suggestion
		newEngineOpts := promql.EngineOpts{
```
seems like a small typo :eye:

Reply:
Nice catch!